src/pages/Dashboard/MintRepayVai/MintVai/index.tsx

```diff
@@ -19,7 +19,7 @@ import { useStyles } from '../styles';
 export interface IMintVaiUiProps {
   disabled: boolean;
   isMintVaiLoading: boolean;
-  mintVai: (value: BigNumber) => Promise<void>;
+  mintVai: (value: BigNumber) => Promise<unknown>;
```

Comment:
This bug was introduced by a previous PR and prevented the tests from running, hence the fix here.